riverdriver/riversqlite/river_sqlite_driver.go

```diff
@@ -499,6 +499,53 @@ func (e *Executor) JobInsertFull(ctx context.Context, params *riverdriver.JobIns
 	return jobRowFromInternal(job)
 }
 
+func (e *Executor) JobInsertFullMany(ctx context.Context, params *riverdriver.JobInsertFullManyParams) ([]*rivertype.JobRow, error) {
+	// TODO: Implement this
+	return nil, nil
```

Comment:
I wasn't totally sure what to do here and didn't want to rabbithole on it while getting the rest of this working. Looks like I just need to loop over single inserts as is done with `JobInsertFastMany`?

Reply:
Yeah, there's no way to do batch insertions right now, so the right answer for this one is just to iterate in the driver and call into the normal `JobInsertFull` a bunch of times.